docs/administration/retentions.md

```diff
@@ -1,7 +1,28 @@
 # Retention policies
 
-!!! tip "Under construction"
+Retention rules serve the purpose of establishing data retention times, specifying when data should be automatically deleted from the platform. Users can define filters to target specific objects, and any objects meeting these criteria that haven't been updated within the designated time frame will be deleted.
```

Comment:
I would make two sentences within the last one to make it easier to read: "[...] specific objects. Any object [...]"
The 's' should also be removed from 'object' if 'any' is used, or 'all objects'.

Reply:
Done